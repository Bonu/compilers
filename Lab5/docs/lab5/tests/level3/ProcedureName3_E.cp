// Main: Test identifier references.

class Main{

    int i2;

    int m =  i2();
    int n =  f();
    int p = OtherClass.f();
    int q = OtherClass.m();
  void main(){
  }
}

class OtherClass {
    int f = Main.i2();;
    int g = Main.f();;

  int m(){
      
  }
}
