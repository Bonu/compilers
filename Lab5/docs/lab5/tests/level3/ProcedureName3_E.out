line 7: i2 is not a method in this context
line 8: unable to resolve name -- f
line 9: OtherClass.f is not a method in this context
line 16: Main.i2 is not a method in this context
line 17: unable to resolve name -- Main.f
5 errors were detected
